graphql/data/schema.js

```diff
@@ -746,6 +749,18 @@ const userType = new GraphQLObjectType({
       description: 'feature values for this specific user',
       resolve: (user, args, context) => getUserFeatures(context, user),
     },
+    searchEngine: {
+      type: SearchEngineType,
+      description: 'the User’s search engine',
+      resolve: (user, args, context) => getUserSearchEngine(context, user),
+    },
+    showYahooPrompt: {
+      type: new GraphQLNonNull(GraphQLBoolean),
+      description: 'whether to show the yahoo search prompt',
+      resolve: user =>
+        !user.yahooSearchSwitchPrompt.hasSeenPrompt &&
+        getUserFeature(YAHOO_SEARCH_EXISTING_USERS).variation,
```

Comment:
Please break this out into its own module. I know it's small, but we try to keep as much business logic out of schema.js as possible to make it more testable.